### ISSUE TITLE
L5.6 "package:discover" Class mailer does not exist

### ISSUE THREAD

**Issue by Cyrille37** [westacks/telebot#25]
Hello

Just a fresh **Laravel 5.6** install, all work.

But `composer require westacks/telebot` throw error:
```
...
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover
In Container.php line 767:
  Class mailer does not exist  
Script @php artisan package:discover handling the post-autoload-dump event returned with error code 1
```

**Comment by Cyrille37** [comment]
Ok, I found this
> This breaks support for 5.6 and below. Method resolved() is only available from Laravel 5.7 onwards.
> at https://github.com/benwilkins/laravel-fcm-notification/pull/40#commitcomment-34699536

I changed for Laravel 5.7 and require is Ok now :-)

**Comment by punyflash** [comment]
Not related to `telebot` library
